release/air_tests/air_benchmarks/workloads/gpu_batch_prediction.py

```diff
@@ -31,8 +31,16 @@ def preprocess(batch: np.ndarray) -> pd.DataFrame:
 
 @click.command(help="Run Batch prediction on Pytorch ResNet models.")
 @click.option("--data-size-gb", type=int, default=1)
-def main(data_size_gb: int):
-    data_url = f"s3://air-example-data-2/{data_size_gb}G-image-data-synthetic-raw"
+@click.option("--smoke-test", is_flag=True, default=False)
+def main(data_size_gb: int, smoke_test: bool = False):
+    data_url = (
+        f"s3://anonymous@air-example-data-2/{data_size_gb}G-image-data-synthetic-raw"
+    )
+
+    if smoke_test:
+        # Only read one image
+        data_url = [data_url + "/dog.jpg"]
```

Comment:
Does this need to be list?

```suggestion
        data_url = data_url + "/dog.jpg"
```

Reply:
Both seem to work fine